### ISSUE TITLE
Claim invitation causes multiple clearnet tabs to open.

### ISSUE THREAD

**Issue by calumcraig** [maidsafe/safe_browser#539]
**Steps to reproduce:**
1. Launch SAFE browser v0.11.1 RC2.
2. Click "Create account".
3. Click "Continue".
4. Click "Claim invitation".

**Expected:**
Clearnet browser is launched at invite.maidsafe.net

**Actual:**
Clearnet URL is opened but multiple instances of the clearnet browser is opened - usually around 6 but higher has been observed too.

![image](https://user-images.githubusercontent.com/44327924/51483744-b6738c00-1d91-11e9-9dbc-46deba986bc3.png)

**Comment by safesurfer** [comment]
confirmed on windows too. opens the URL infinitely on clearnet browser until you close the tab (which is opened when you click `Claim Invitation`) in the safe_browser.
@calumcraig @joshuef

**Comment by calumcraig** [comment]
Retesting this on Linux Mint 19 - RC3 of the SAFE Browser has resolved this issue.

When clicking on the clearnet link referenced above to claim an invitation the SAFE browser opens one new tab which then closes as the clearnet browser opens.

The behaviour where we saw multiple instances of the clearnet browser has been resolved:
![image](https://user-images.githubusercontent.com/44327924/51609014-f0b76780-1f10-11e9-85bb-8cb3e5a0e8c7.png)

QA complete, moving this issue to done.
